publish.sh

```diff
@@ -111,7 +111,7 @@ echo "Generating production data"
 
 STARTTIME=${SECONDS}
 
-hdfs_imagerec_prod=/user/${username}/imagerec_prod/data
```

Comment:
I made this change following the format from imagerec but not sure if it should be kept

Reply:
Nice one! It's much better to keep things standardized and predictable.